### PR TITLE
Fix user profile bloc unit tests

### PR DESCRIPTION
**Pull request by ademar111190**
Fixes https://github.com/breez/c-breez/issues/459

I'm adding a "UnitLogTree" to take advantage of fimber on unit tests; it helped me a lot on these tests.

### CODE REVIEW

**Review by erdemyerebasmaz — APPROVED** [review]
LGTM

**Review by ubbabeck — APPROVED** [review]
LGTM too 🚀

File: test/unit_logger.dart [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+  @override
+  void log(
+    String level,
+    String message, {
+    String? tag,
+    Object? ex,
+    StackTrace? stacktrace,
+  }) {
+    print("$tag [$level] $message");
```
Comment (ubbabeck) [inline test/unit_logger.dart:17]: the dart analyser tool does not like when we use print in production

File: test/unit_logger.dart [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    if (ex != null) print(ex);
+    if (stacktrace != null) print(stacktrace);
```
Comment (ubbabeck) [inline test/unit_logger.dart:19]: same here

File: test/unit_logger.dart [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  @override
+  void log(
+    String level,
+    String message, {
+    String? tag,
+    Object? ex,
+    StackTrace? stacktrace,
+  }) {
+    print("$tag [$level] $message");
+    if (ex != null) print(ex);
```
Comment (ubbabeck) [inline test/unit_logger.dart:18]:
```
info • test/unit_logger.dart:18:21 • Don't invoke 'print' in production
          code. Try using a logging framework. • avoid_print
```

**Review by ubbabeck — CHANGES_REQUESTED** [review]
Do you think the workflow build will success when these are removed?

**Review by erdemyerebasmaz — APPROVED** [review]

**Review by ubbabeck — APPROVED** [review]

**Pull request merged**